src/app/inventory/Inventory.tsx

```diff
@@ -69,6 +72,7 @@ class Inventory extends React.Component<Props> {
     return (
       <>
         <Stores />
+        {!mobile && <StJudeToaster />}
```

Comment:
We also have an `isPhonePortrait` in redux that can be used to detect phone mode.